md/deploy-bundle.md

```diff
@@ -23,10 +23,89 @@ It is useful to update the configuration, locally or from a remote computer.
 
 1. Download the deploy.zip file from the [Bonitasoft download page](http://www.bonitasoft.com/downloads-v2) for the Community edition 
 or from the [Customer Portal](https://customer.bonitasoft.com/download/request) for Subscription editions.
-2. Unzip the `deploy.zip` into a temporary folder.
-3. Install Bonita BPM in your Application Server, by merging the appropriate folder (e.g. `Wildfly-10.1.0.Final` or `Tomcat-7.0.67`) from `BonitaBPMSubscription-x.y.z-deploy` folder
-into the existing root folder of your application server.
-4. Follow the [Tomcat bundle steps](tomcat-bundle.md#configuration) or [WildFly bundle steps](wildfly-bundle.md#configuration)
+2. Unzip the `deploy.zip` into a temporary folder (DEPLOY_ZIP_HOME).
+4. Follow the [Tomcat bundle steps](#tomcat-installation) or [WildFly bundle steps](#wildfly-installation)
+
+<a id="tomcat-installation" />
+
+## Tomcat installation
+### Install Bonita BPM Platform in Tomcat
+
+Copy all files and directories from DEPLOY_ZIP_HOME/Tomcat-7.0.76/server to your Tomcat root directory (TOMCAT_HOME).
+:::warning
```

Comment:
Please ignore my original comment here too, since I have now verified the rendering in the docker instance.